hazelcast/src/main/java/com/hazelcast/sql/SqlService.java

```diff
@@ -89,6 +89,42 @@ default SqlResult execute(@Nonnull String sql, Object... arguments) {
         return execute(statement);
     }
 
+    /**
+     * Convenient method to execute a distributed query with the given
+     * parameter values.
+     * <p>
+     * Converts passed SQL string and parameter values into an {@link
+     * SqlStatement} object and invokes {@link #execute(SqlStatement)}.
+     *
+     * <p>
+     * This method is meant to be used for statements other than "SELECT" queries and only if user does not really need the
+     * count of updated entries.
+     *
+     * @param sql       SQL string
+     * @param arguments query parameter values that will be passed to {@link SqlStatement#setParameters(List)}
+     * @throws NullPointerException     if the SQL string is null
+     * @throws IllegalArgumentException if the SQL string is empty
+     * @throws HazelcastSqlException    in case of execution error
+     * @see SqlService
+     * @see SqlStatement
+     * @see #execute(SqlStatement)
+     *
+     * @since 5.3
+     */
+    @SuppressWarnings({"EmptyTryBlock", "checkstyle:EmptyBlock"})
+    default void executeStatement(@Nonnull String sql, Object... arguments) {
```

Comment:
I do not like the name

Reply:
I mean, even if it does, those will be simply discarded. Side effects in SQLa are rare but also possible.

I don't have an idea for better name. Maybe `runCommand`?